generators/languages/templates/src/main/webapp/i18n/fr/_logs.json

```diff
@@ -8,6 +8,7 @@
             "level": "Niveau"
         }
     },
+    <%_ if (databaseType !== 'no' && databaseType !== 'cassandra') { _%>
```

Comment:
This file has this by mistake, as we have a dedicated json for this, hence there was no need to rename logs.json

Reply:
i'm fixing it